metadata-ingestion/tests/integration/snowflake/test_snowflake_classification.py

```diff
@@ -0,0 +1,99 @@
+from functools import partial
+from typing import cast
+from unittest import mock
+
+import pandas as pd
+import pytest
+
+from datahub.configuration.common import AllowDenyPattern, DynamicTypedConfig
+from datahub.ingestion.glossary.classifier import (
+    ClassificationConfig,
+    DynamicTypedClassifierConfig,
+)
+from datahub.ingestion.glossary.datahub_classifier import DataHubClassifierConfig
+from datahub.ingestion.run.pipeline import Pipeline
+from datahub.ingestion.run.pipeline_config import PipelineConfig, SourceConfig
+from datahub.ingestion.source.snowflake.snowflake_config import SnowflakeV2Config
+from datahub.ingestion.source.snowflake.snowflake_report import SnowflakeV2Report
+from tests.integration.snowflake.common import default_query_results
+
+NUM_SAMPLE_VALUES = 100
+
+sample_values = ["abc@xyz.com" for _ in range(NUM_SAMPLE_VALUES)]
+
+
+# Uncomment below skip line to run this test. The test takes approx 4.5 minutes.
+# Run with --durations=0 to show the timings for different combinations
+# @pytest.mark.skip
```

Comment:
we shouldn't be running this full thing in all tests

maybe we can do a pytest.mark.skipif mark on the test cases, so if `DATAHUB_TEST_CLASSIFY_PERFORMANCE` is set then it runs the full thing, otherwise it only runs one test case config

Reply:
Right. I'll fix this. I'd planned to mark this as to be skipped always, but what you are suggesting sounds better to me.